backend/eslint.config.js

```diff
@@ -0,0 +1,61 @@
+//
+// SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+const neostandard = require('neostandard')
+const pluginJest = require('eslint-plugin-jest')
+const pluginSecurity = require('eslint-plugin-security')
+const pluginLodash = require('eslint-plugin-lodash')
+const pluginImport = require('eslint-plugin-import')
+
+module.exports = [
+  ...neostandard({}),
+  {
+    rules: {
+      '@stylistic/comma-dangle': ['error', 'always-multiline'],
+      'no-console': 'error',
+      'no-debugger': 'error',
```

Comment:
still need since we have maxWarnings=0?

Reply:
I have removed `no-debugger` since it is already part of the neostandard but we still need `no-console`